lib/paper_trail/events/create.rb

```diff
@@ -20,7 +20,9 @@ def data
           data[:created_at] = @record.updated_at
         end
         if record_object_changes? && changed_notably?
-          data[:object_changes] = recordable_object_changes(changes)
+          changes = serialize_changes(notable_changes)
+          changes = recordable_object_changes(changes)
+          data[:object_changes] = changes
```

Comment:
I'm concerned this changes the order of operations. Previously, `object_changes_adapter` happened first, before `AttributeSerializers::ObjectChangesAttribute`. Now it happens after. Was that intentional?

Reply:
Hmm, that's not how I'm reading it. Previously we had this code:
```ruby
data[:object_changes] = recordable_object_changes(changes)
```
`changes` used to find the notable changes and serialize them. then `recordable_object_changes`, the last operation, is what called `object_changes_adapter`.